configuration/locations/locations.csv

```diff
@@ -1,34 +1,29 @@
-7b4282ff-5575-4079-a847-d275d68831fe,,LOCATION_MATERNITY_WARD_ROOM,LOCATION_MATERNITY_WARD_ROOM,LOCATION_LABOR_WARD,Admission Location,,,,,,,,,,,,
+8e203517-f462-406c-bdf9-12590e28ccf4, ,LOCATION_HOSPITAL,LOCATION_HOSPITAL,,Visit Location,CENTRE,MFOUNDI,ARRONDISSEMENT DE YAOUNDE II,YAOUNDE II,YAOUNDE VILLE,CITE VERTE,,CAMEROON,,,,
+0efa13f3-4351-4e9c-91fd-1d33a1833028, ,LOCATION_ART,LOCATION_ART,LOCATION_HOSPITAL,Login Location;Appointment Location,,,,,,,,CAMEROON,,,,
+1144b765-2e6c-47b9-bb2d-e059ad1680a8, ,LOCATION_ANC,LOCATION_ANC,LOCATION_HOSPITAL,Login Location;Appointment Location,,,,,,,,CAMEROON,,,,
+ff553570-8eca-4649-9c44-e1b06921796c, ,LOCATION_ART_DISPENSATION,LOCATION_ART_DISPENSATION,LOCATION_HOSPITAL,Login Location;Appointment Location,,,,,,,,CAMEROON,,,,
+0d9bf0c2-39f5-489a-a660-927f1ac97b7b, ,LOCATION_CARDIOLOGY,LOCATION_CARDIOLOGY,LOCATION_HOSPITAL,Login Location;Appointment Location,,,,,,,,CAMEROON,,,,
+c6e23237-5a0d-438a-9381-ac403d30054f, ,LOCATION_DELIVERY,LOCATION_DELIVERY,LOCATION_HOSPITAL,Login Location;Appointment Location,,,,,,,,CAMEROON,,,,
+79157d5d-1e45-407d-bf15-ae8dc1674a70, ,LOCATION_DENTISTRY,LOCATION_DENTISTRY,LOCATION_HOSPITAL,Login Location;Appointment Location,,,,,,,,CAMEROON,,,,
+d1757c49-d3eb-4b36-bb2c-1d3faf238b20, ,LOCATION_EMERGENCY,LOCATION_EMERGENCY,LOCATION_HOSPITAL,Login Location,,,,,,,,CAMEROON,,,,
+4f2e70b3-9b18-4773-994e-c67b31422722, ,LOCATION_FAMILY_PLANNING,LOCATION_FAMILY_PLANNING,LOCATION_HOSPITAL,Login Location;Appointment Location,,,,,,,,CAMEROON,,,,
+8568c575-16fc-4330-a3bb-c0460c15a298, ,LOCATION_GYNAECOLOGY,LOCATION_GYNAECOLOGY,LOCATION_HOSPITAL,Login Location;Appointment Location,,,,,,,,CAMEROON,,,,
+31b46228-0cb2-4e33-b8aa-4fcbae016adc, ,LOCATION_KINESIOLOGY,LOCATION_KINESIOLOGY,LOCATION_HOSPITAL,Login Location;Appointment Location,,,,,,,,CAMEROON,,,,
+ee0644c1-53e1-4c8c-bcd9-2308c608be60, ,LOCATION_LABORATORY,LOCATION_LABORATORY,LOCATION_HOSPITAL,Login Location;Appointment Location,,,,,,,,CAMEROON,,,,
+ff6b443b-958b-4cf9-b187-d0f6da6b6d94, ,LOCATION_MAJOR_SURGERY,LOCATION_MAJOR_SURGERY,LOCATION_HOSPITAL,Login Location;Appointment Location,,,,,,,,CAMEROON,,,,
+0d414ccb-55a2-480f-93ff-c2af7061f0ba, ,LOCATION_MATERNITY,LOCATION_MATERNITY,LOCATION_HOSPITAL,Login Location;Appointment Location,,,,,,,,CAMEROON,,,,
+0607712c-aced-44da-8534-72d639bd9994, ,LOCATION_MINOR_SURGERY,LOCATION_MINOR_SURGERY,LOCATION_HOSPITAL,Login Location;Appointment Location,,,,,,,,CAMEROON,,,,
+4863dd00-0543-40c1-9cb0-f019c11e86bf, ,LOCATION_NEONATOLOGY,LOCATION_NEONATOLOGY,LOCATION_HOSPITAL,Login Location;Appointment Location,,,,,,,,CAMEROON,,,,
+4dd3bdc2-38ab-450e-b36f-784ed6f8427d, ,LOCATION_OPD,LOCATION_OPD,LOCATION_HOSPITAL,Login Location;Appointment Location,,,,,,,,CAMEROON,,,,
+df204697-a6c6-4768-a4e8-7082940fc8eb, ,LOCATION_OPTHALMOLOGY,LOCATION_OPTHALMOLOGY,LOCATION_HOSPITAL,Login Location;Appointment Location,,,,,,,,CAMEROON,,,,
```

Comment:
@prashantsable1991 There is a small typo here, the right spelling is:  OPHTHALMOLOGY